app/controllers/agent/ConfirmClientUTRController.scala

```diff
@@ -49,6 +52,14 @@ class ConfirmClientUTRController @Inject()(confirmClient: confirmClient,
 
   def submit: Action[AnyContent] = Authenticated.async { implicit request =>
     implicit user =>
+      auditingService.extendedAudit(ConfirmClientDetailsAuditModel(
```

Comment:
assuming that clients details after successful login will be in the context, we can change next code to something like:
```
for {
        clientName <- fetchClientName
        nino <- request.session.get(SessionKeys.clientNino)
        clientMTDID <- request.session.get(SessionKeys.clientMTDID)
        arn <- user.agentReferenceNumber
        saUtr <- request.session.get(SessionKeys.clientUTR)
      } yield
        auditingService.extendedAudit(ConfirmClientDetailsAuditModel(
          clientName = clientName,
          nino = nino,
          mtditid = clientMTDID,
          arn = arn,
          saUtr = saUtr,
          credId = user.credIdd
        )
```